README.md

```diff
@@ -106,6 +107,17 @@ The Akita is a Japanese dog breed known for its loyalty and dedication. It's als
 
 We were also very inspired by the story of [Hachikō](https://en.wikipedia.org/wiki/Hachik%C5%8D) - a dog of immense loyalty. Hachikō was an Akita who would meet with his owner every day at the train station. Even after his owner passed away, Hachikō continued to wait for his owner at the train station for more than 9 years. Hachikō continues to have a legacy in Japanese culture, with various statues created in his memory - there was even an [American film](https://en.wikipedia.org/wiki/Hachi:_A_Dog%27s_Tale) based on him.
 
+### Akita Data
+Akita collects data on
+
+- your time spent and number of visits on all websites,
+- the Payment Pointers present on the websites you visit,
+- and the currency and amount of payments streamed to [Payment Pointers](https://paymentpointers.org/) via Web Monetization.
```

Comment:
I think the link to Payment Pointers would be better for the previous bullet point where they're first mentioned